### ISSUE TITLE
Elementor Pro: Media Carousel Warning in PHP 7.4.5

### ISSUE THREAD

**Issue by peca993** [elementor/elementor#11355]
When i updated PHP version to **7.4.5** i got a waring: 
T**rying to access array offset on value of type null** 
in file **/elementor-pro/modules/carousel/widgets/media-carousel.php** 
at line **243** 
in function **get_image_link_to**

**Comment by markc** [comment]
Same problem here but with PHP 7.4.3-4ubuntu2.2, WP 5.4.1 with Elementor Pro v2.9.11/2.9.5
```
Trying to access array offset on value of type null in /*/wp-content/plugins/elementor-pro/modules/carousel/widgets/media-carousel.php on line 243
Trying to access array offset on value of type bool in /*/wp-content/plugins/elementor/includes/managers/image.php on line 88
```

**Comment by osaulnier** [comment]
Seems it doesn't rely to the PHP version.
It breaks on this line 243 : if ( $slide['video']['url'] ) {
when the slide is an image and not a video the array $slide['video'] is null.
The test 'video' === $slide['type'] is missing like in line 293 where it exists
Btw the next line 244 should be return $slide['video']['url'] to be clearer

**Comment by erfan-ilyas** [comment]
I am also having this issue. Please fix. +1

**Comment by narkan** [comment]
Also having issue

**Comment by rodolphebertozzo** [comment]
Please fix. +1

**Comment by RissaFP15** [comment]
Also have this issue

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v3.0.0

Feel free to update

Thanks!
